lib/puppet/type/check_http.rb

```diff
@@ -11,7 +11,7 @@
 
 Use this to check whether a web server is responding correctly. This can be used both as a prerequisite (don't manage something if a dependency is unhealthy) or to check whether everything went right after managing something.
 EOS
-  features: [],
+  features: ['custom_insync'],
   attributes: {
     ensure: {
       type: 'Enum[present, absent]',
```

Comment:
Please remove the `ensure` attribute, as the custom_insync feature now doesn't require any properties anymore.